.env

```diff
@@ -1,16 +1,25 @@
 # This file is commited. Do not store secrets here
 
-REACT_APP_ENV=staging
-REACT_APP_QUERY_NODE_URL=https://sumer-dev-2.joystream.app/query/server/graphql
-REACT_APP_QUERY_NODE_SUBSCRIPTION_URL=wss://sumer-dev-2.joystream.app/query/server/graphql
-REACT_APP_ORION_URL=https://orion-staging.joystream.app/graphql
-REACT_APP_NODE_URL=wss://sumer-dev-2.joystream.app/rpc
-REACT_APP_FAUCET_URL=https://sumer-dev-2.joystream.app/members/register
+REACT_APP_ENV=development
 
-#REACT_APP_ENV=production
-#REACT_APP_QUERY_NODE_URL=https://hydra.joystream.org/graphql
-#REACT_APP_QUERY_NODE_SUBSCRIPTION_URL=wss://hydra.joystream.org/graphql
-#REACT_APP_ORION_URL=https://orion.joystream.org/graphql
-#REACT_APP_NODE_URL=wss://rome-rpc-endpoint.joystream.org:9944/
-#REACT_APP_FAUCET_URL=https://member-faucet.joystream.org/register
+# default is staging
+REACT_APP_STAGING_QUERY_NODE_URL=https://sumer-dev-2.joystream.app/query/server/graphql
```

Comment:
On a second thought, maybe let's call this env `development`? Staging would be actually a separate one, like we used `reckless` for staging before `sumer`.